Plugins/GeoModel/src/GeoModelMaterialConverter.cpp

```diff
@@ -0,0 +1,39 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/GeoModel/GeoModelMaterialConverter.hpp"
+
+#include "Acts/Material/Material.hpp"
+
+#include "GeoModelKernel/GeoMaterial.h"
+#include "GeoModelKernel/Units.h"
+namespace {
+constexpr double s_densityCnvFactor = 1. / GeoModelKernelUnits::gram;
+}
+Acts::Material Acts::GeoModel::geoMaterialConverter(const GeoMaterial* gm,
```

Comment:
```suggestion
#include "GeoModelKernel/Units.h"

namespace {
constexpr double s_densityCnvFactor = 1. / GeoModelKernelUnits::gram;
}

Acts::Material Acts::GeoModel::geoMaterialConverter(const GeoMaterial* gm,
```

also: might be cleaner to define this conversion constant in the function below

Reply:
moved into the function